easybuild/easyblocks/g/gromacs.py

```diff
@@ -182,6 +182,12 @@ def sanity_check_step(self):
         if self.toolchain.options.get('usempi', None):
             suff = '_mpi'
 
+        # Add the _d suffix to the suffix, in case of the double precission
+        if '-DGMX_DOUBLE=on' in self.cfg['configopts']:
+            suff = suff + '_d'
+        elif '-DGMX_DOUBLE=ON' in self.cfg['configopts']:
+            suff = suff + '_d'
```

Comment:
I realized that I did not answer the question :-)

One other option could be to check directly the CMake cache (after the configure step):
```
cmake -LA . | grep 'GMX_DOUBLE:BOOL=ON'
```
or (maybe less portable)
```
grep 'GMX_DOUBLE:BOOL=ON' CMakeCache.txt
```
I believe it should always be `ON` or `OFF` in there.